requirements.txt

```diff
@@ -10,3 +10,4 @@ canvasapi==0.15.0
 beautifulsoup4==4.8.2
 dateparser==0.7.4
 pyyaml==5.3.1
+pyjwt
```

Comment:
Let's pin this?

Reply:
Also, I wouldn't complain if you combined with the others and alphabetized :)

Reply:
Yeah, I remembered I wanted to do this last night but it was late when this finished, thanks.